client/notifications/index.jsx

```diff
@@ -53,6 +55,25 @@ const isDesktop = config.isEnabled( 'desktop' );
 
 const debug = debugFactory( 'notifications:panel' );
 
+const Notifications3PCNotice = ( { className } ) => {
+	const translate = useTranslate();
+	return (
+		<div className={ `reader-notifications__3pc-notice ${ className }` }>
+			<p>
+				{ translate(
+					"Didn't expect to see this page? {{learnMoreLink}}Learn about 3rd party cookies.{{/learnMoreLink}}",
```

Comment:
I saw this when testing RTL, but it could just be that it needs translation:

<img width="494" alt="Screenshot 2025-04-04 at 2 01 13 PM" src="https://github.com/user-attachments/assets/2ffdb26d-0bbb-49f8-aa9d-481990bf27a0" />

The period moved to the beginning.

Reply:
Interesting. I assume thats a translation issue.